add snapshot info (#138)
DefangLabs/defang-docs: docs/concepts/managed-storage/managed-redis.md

    - mode: host
      target: 6379
```

## Final Snapshots

> When a project is deployed to a production environment any managed redis instances are automatically

Reviewer: ```suggestion
When a project is deployed with the `production` [deployment mode](/docs/concepts/deployment-modes) any managed redis instances are automatically
```
